src/core/ui/forms/FormikFileInput/FormikFileInput.tsx

```diff
@@ -4,19 +4,36 @@ import FormikInputField, { FormikInputFieldProps } from '../FormikInputField/For
 import FileUploadButton from '../../upload/FileUpload/FileUpload';
 import { useStorageConfigContext } from '../../../../domain/platform/storage/StorageBucket/StorageConfigContext';
 
+const DEFAULT_PROTOCOL = 'https';
+const MATCH_PROTOCOL_REGEX = /^[a-z][a-z0-9+_-]{0,500}:\/\//i;
+
 type FormikFileInputProps = FormikInputFieldProps & {
   referenceID?: string;
+  defaultProtocol?: string;
 };
 
-const FormikFileInput = ({ name, referenceID, ...props }: FormikFileInputProps) => {
-  const [, , helpers] = useField(name);
+const FormikFileInput = ({ name, referenceID, defaultProtocol = DEFAULT_PROTOCOL, ...props }: FormikFileInputProps) => {
+  const [field, , helpers] = useField(name);
 
   const storageConfig = useStorageConfigContext();
 
+  const checkProtocol = () => {
+    if (!defaultProtocol) {
+      return;
+    }
+    if (field.value) {
+      const currentValue = `${field.value}`; // make sure field.value is a string
```

Comment:
Is this just casting to string?

Reply:
not exactly, field.value is `any`. this is more like a `toString()` to convert anything that comes to string

Reply:
Not sure if we need the comment. Maybe is confusing but however you think is ok.